lib/pg_query/deparse.rb

```diff
@@ -243,13 +245,14 @@ def deparse_raw_stmt(node)
 
     def deparse_renamestmt(node)
       output = []
-
       case node['renameType']
-      when OBJECT_TYPE_TABLE
-        output << 'ALTER TABLE'
+      when OBJECT_TYPE_TABLE, OBJECT_TYPE_VIEW
+        output << 'ALTER'
+        output << 'TABLE' if node['renameType'] == OBJECT_TYPE_TABLE
+        output << 'VIEW' if node['renameType'] == OBJECT_TYPE_VIEW
         output << deparse_item(node['relation'])
         output << 'RENAME TO'
-        output << node['newname']
+        output << deparse_identifier(node['newname'], escape_always: true)
```

Comment:
Can you clarify why we need to escape this one always?

I mostly added that as a legacy option to avoid breaking too many tests, but I think going forward this should not be used if we can avoid it.

Reply:
ALTER VIEW "foo" RENAME TO "Bar"
if execute this sql without (escape_always: true) options, deparser result is `ALTER VIEW "foo" RENAME TO Bar`. If you execute deparser sql, rename the view as a bar not Bar.
So we need this options.